lib/install.js

```diff
@@ -105,7 +106,10 @@ function place_binary(from,to,opts,callback) {
         });
 
         req.on('response', function(res) {
-            if (res.statusCode !== 200) {
+           if (res.statusCode === 302) {
```

Comment:
You probably want to allow other 30x codes here too I think.

Reply:
I've changed it to check for a Location: header instead.